apps/api/v1/pages/api/bookings/_get.ts

```diff
@@ -180,8 +180,10 @@ export async function handler(req: NextApiRequest) {
   const { dateFrom, dateTo } = schemaBookingGetParams.parse(req.query);
 
   const args: Prisma.BookingFindManyArgs = {};
-  args.take = take;
-  args.skip = skip;
+  if (req.query.take && req.query.page) {
+    args.take = take;
+    args.skip = skip;
+  }
```

Comment:
Makes pagination optional and thus non-breaking

Reply:
Should this be checking req.query.take or just simply take that's passed into the function already?

Reply:
Take that's already passed in the function are fallbacks (defaults). If we check them, then this check will not work as expected. I wonder if defaults are even needed though, because pagination in this way would only work when take and page are explicitly passed by the user.

Reply:
Ah I see. ok.